catalogs/catalog-hadoop/src/test/java/org/apache/gravitino/catalog/hadoop/integration/test/HadoopCatalogIT.java

```diff
@@ -557,6 +557,7 @@ public void testFilesetRemoveProperties() throws IOException {
   }
 
   @Test
+  @SuppressWarnings("deprecation")
```

Comment:
you should update the test code to use the new API instead of adding `SuppressWarnings` annotation